templates/ts/copilot-plugin-from-scratch/env/.env.dev.user

```diff
@@ -0,0 +1,5 @@
+# This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project.
```

Comment:
Do we need to commit this user data file to template, or it will be auto generated by running?

Reply:
I checked out other templates and noticed that they also submitted the user data file. Maybe we should do the same for now.